spyder_remote_services/services/__init__.py

```diff
@@ -1,3 +1,4 @@
 from spyder_remote_services.services.envs_manager.handlers import handlers as envs_manager_handlers
+from spyder_remote_services.services.fsspec.handlers import handlers as fsspec_handlers
```

Comment:
Given that `fsspec` is not going to be used at the end to implement this functionality, I suggest to rename this simply to `fs_handlers` to avoid confusions.